src/sentry/models/files/abstractfile.py

```diff
@@ -107,6 +107,7 @@ def fetch_file(offset, getfile):
                 exe.submit(fetch_file, idx.offset, idx.blob.getfile)
 
         mem.flush()
+        mem.close()
```

Comment:
mmap supports the context manager protocol so I would recommend that instead

mmap also doesn't trigger a `ResourceWarning` when left unclosed (though arguably it should) so your warnings are definitely coming from somewhere else -- do you have some sample lines?

Reply:
yeah, i think its coming from the getfile function that is passed in (https://github.com/getsentry/getsentry/blob/056e9cc337b98984b37ffd3b40a79adea07c32e5/getsentry/filestore.py#L141 in prod)
and example warning is: 
```
/usr/src/sentry/src/sentry/models/files/abstractfile.py:38: ResourceWarning: unclosed <socket.socket fd=179, family=2, type=1, proto=6, laddr=('10.4.161.44', 43014), raddr=('10.8.0.230', 11211)>
  self._prefetch(prefetch_to, delete)
```